src/getting-started/contribute.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Contribute to the Docs
+---
+
+Our User Guide is open source and hosted in a [public-facing repository on GitHub][1], where you can submit changes through pull requests (PRs). This open source approach to documentation has many advantages:
+
+- An open and transparent process to cultivate the most helpful content about using Magento
+- Continuous updates and publishing for content improvements and changes
+- Ongoing collaboration between between customers, partners, our product teams, and our documentation team
+
+## Working with collaborative documentation
+
+As a Magento user - regardless of whether you are an employee, partner, customer, or even a prospective customer - you have the opportunity of contributing to this documentation in several simple ways:
+
+- Provide feedback about helpfulness of the page
+- Log an issue against a specific page
+- Submit a quick edit where you see basic errors
+- Authoring entire pages, complete with graphics
+
+These sections outline everything you need to know to interact with and contribute to this User Guide.
+
+## Provide feedback
+
+All User Guide pages display a feedback icon.
+
+![Click the feedback icon]({% link getting-started/assets/feedback-icon.png %})
+
+Simply click this icon to provide a rating for the current page and, optionally, give us some actionable feedback about how we can make it better.
+
+## Log an issue
+
+When you see a problem with one of our pages, please let us know. This could be a broken link, content error, or other item that you can identify for fixing.
```

Comment:
not a fan of please in docs.  consider removing

Reply:
I don't like it  in our standard stuff either.  But in this case we are asking them for something. But I would totally remove from our product information or tutorials.